Example/AppDelegate.swift

```diff
@@ -12,8 +13,20 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     @available(iOS 12.0, *)
     lazy var carPlaySearchController: CarPlaySearchController = CarPlaySearchController()
 
+    @available(iOS 12.0, *)
+    lazy var interfaceController: CPInterfaceController? = nil
+
+    @available(iOS 12.0, *)
+    lazy var carWindow: CPWindow? = nil
+
+    @available(iOS 12.0, *)
+    lazy var sessionConfiguration: CPSessionConfiguration? = nil
+
```

Comment:
@avi-c, do we really need these properties in `AppDelegate.swift`?

Reply:
Good catch, these properties appear to be unused, if I’m not mistaken.